feat(sep): install DeputyPauseModule into FND ops safe (#495)
ethereum-optimism/superchain-ops: tasks/sep/029-deputy-pause-module/README.md

Reviewer: NIT: Add that the predeployement is made with version: `1.0.0-beta.2`. 
that is currently availaible at this link
-> https://github.com/ethereum-optimism/optimism/blob/cf7a37b6b9f46e259b4ecf5c709f465f63a5e0fd/packages/contracts-bedrock/src/safe/DeputyPauseModule.sol#L90

# DeputyPauseModule Installation - OP Sepolia

Status: DRAFT, NOT READY TO SIGN

## Objective

Installs the `DeputyPauseModule` into the Optimism Foundation Operations Safe for OP Sepolia.

## Pre-deployments

- `DeputyPauseModule` - `0x62f3972c56733aB078F0764d2414DfCaa99d574c`

## Simulation

Please see the "Simulating and Verifying the Transaction" instructions in [SINGLE.md](../../../SINGLE.md).
When simulating, ensure the logs say `Using script /your/path/to/superchain-ops/tasks/sep/029-deputy-pause-module/SignFromJson.s.sol`.
This ensures all safety checks are run. If the default `SignFromJson.s.sol` script is shown (without the full path), something is wrong and the safety checks will not run.

## State Validation

> Please see the instructions for [validation](../base-005-fp-holocene-upgrade/VALIDATION.md).

Reviewer: The validation is incorrect here 
```suggestion
Please see the instructions for [validation](../029-deputy-pause-module/VALIDATION.md).
```

## Execution

This upgrade:

> - Installs the `DeputyPauseModule` into the Optimism Foundation Operations Safe.

Reviewer: ```suggestion
- Installs the `DeputyPauseModule` into the Optimism Foundation Operations Safe that is [deployed here](https://sepolia.etherscan.io/address/0x62f3972c56733aB078F0764d2414DfCaa99d574c#code)
```

# DeputyPauseModule Installation - OP Sepolia

> Status: DRAFT, NOT READY TO SIGN

Reviewer 2: I think this will be ready to sign once we merge?
```suggestion
Status: READY TO SIGN
```
